### PR TITLE
config map for AddressConfig

### PR DESCRIPTION
**Pull request by microArtur**
https://nanopay.atlassian.net/browse/NP-8818

### CODE REVIEW

File: src/foam/nanos/auth/AddressConfig.js [hunk 1]
```diff
@@ -30,6 +34,17 @@
     {
       class: 'Int',
       name: 'suite'
+    },
+    {
+      class: 'Array',
+      of: 'foam.nanos.auth.ViewPropertyConfig',
+      name: 'propertyConfig'
```
Comment (kgrgreer) [inline src/foam/nanos/auth/AddressConfig.js]: Could you add documentation:

File: src/foam/nanos/auth/ViewPropertyConfig.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.auth',
```
Comment (jlhughes) [inline src/foam/nanos/auth/ViewPropertyConfig.js]: This should not be in auth. 
Perhaps u2/view 
@kgrgreer ?

File: src/foam/nanos/auth/AddressConfig.js (hunk 1 above)
Comment (Nauna) [reply src/foam/nanos/auth/AddressConfig.js]: Agree, so this is just configuring the properties of the first row in address view. It should be limited to self.Address.STREET_NUMBER, self.Address.STREET_NAME, self.Address.SUITE and should mention that it only takes into account visibility hidden and order.

I wonder if you can try:
return this.propertyConfig.map( v => this.Address.getAxiomByName(v.propertyName).copyFrom(v));

Then somewhere between here and the address view you make 'var data = Address.properties = addressConfig.getProperties();
Then the view just displays data in a SectionedDetailView

File: src/foam/nanos/auth/ViewPropertyConfig.js (hunk 2 above)
Comment (jlhughes) [reply src/foam/nanos/auth/ViewPropertyConfig.js]: @microArtur move to u2/view as this is generic.

File: src/foam/nanos/auth/ViewPropertyConfig.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  name: 'ViewPropertyConfig',
```
Comment (sarthak-marwaha) [inline src/foam/nanos/auth/ViewPropertyConfig.js]: Does this even need to be modelled? We typically just use a Map of Maps for this behaviour as seen in [VerticalDetailView ](https://github.com/kgrgreer/foam3/blob/5ea78878ebff1f195210448bbe88f25638b21335/src/foam/u2/detail/VerticalDetailView.js#L18-L22) So in this case this model can be discarded and the AddressConfig model can have only two properties for countryId and config Map property

Comment (jlhughes) [reply src/foam/nanos/auth/ViewPropertyConfig.js]: This was intentional, to capture additional 'view' behaviour going forward.

**Review by jlhughes — APPROVED** [review]

**Review by sarthak-marwaha — APPROVED** [review]

**Pull request merged**